File: Oke_client/Oke_teacher/Oke_teacher/Entity/Mail.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Oke_teacher.Entity
{
    /// <summary>
    /// 投稿信息封装类
    /// </summary>
    public class Mail
    {
        //邮件ID
        public int mailId { set; get; }
        //邮件主题
        public string mailTitle { set; get; }
        //邮件描述
        public string mailDescribe { set; get; }
        //邮件发起方
        public User mailFromUser { set; get; }
        //邮件接收方
        public User mailToUser { set; get; }
    }
}

File: Oke_server/oke/src/main/java/com/bnuz/oke/entity/Mail.java
package com.bnuz.oke.entity;

import java.io.Serializable;

/**
 * Mail
 *
 * @author handsome
 * @date 2020年 06月09日 11:11:53
 */
public class Mail implements Serializable {

	private static final long serialVersionUID = 2113025072512738080L;

	private int mailId;

	private String mailTitle;

	private String mailDescribe;

	private User mailFromUser;

	private User mailToUser;

	public int getMailId() {
		return mailId;
	}

	public void setMailId(int mailId) {
		this.mailId = mailId;
	}

	public String getMailTitle() {
		return mailTitle;
	}

	public void setMailTitle(String mailTitle) {
		this.mailTitle = mailTitle;
	}

	public String getMailDescribe() {
		return mailDescribe;
	}

	public void setMailDescribe(String mailDescribe) {
		this.mailDescribe = mailDescribe;
	}

	public User getMailFromUser() {
		return mailFromUser;
	}

	public void setMailFromUser(User mailFromUser) {
		this.mailFromUser = mailFromUser;
	}

	public User getMailToUser() {
		return mailToUser;
	}

	public void setMailToUser(User mailToUser) {
		this.mailToUser = mailToUser;
	}

	@Override
	public String toString() {
		return "Mail{" +
				"mailId=" + mailId +
				", mailTitle='" + mailTitle + '\'' +
				", mailDescribe='" + mailDescribe + '\'' +
				", mailFromUser=" + mailFromUser +
				", mailToUser=" + mailToUser +
				'}';
	}
}
